### ISSUE TITLE
About the expert policy

### ISSUE THREAD

**Issue by RayYoh** [johnrso/spawnnet#2]
Dear authors,
Thanks for your excellent work.
I want to ask where and how we can get this expert policy.
![image](https://github.com/johnrso/spawnnet/assets/35106336/1e50494d-289e-40c0-ac57-7542e46ad960)

**Comment by sashwat-mahalingam** [comment]
Hello,

Thank you for your inquiry. Please refer to the [RLAfford README subsection about the expert policy](https://github.com/johnrso/spawnnet/blob/main/RLAfford/README.md) as it contains a link to a Google Drive folder with all the expert policies.

**Comment by RayYoh** [comment]
> Hello,
> 
> Thank you for your inquiry. Please refer to the [RLAfford README subsection about the expert policy](https://github.com/johnrso/spawnnet/blob/main/RLAfford/README.md) as it contains a link to a Google Drive folder with all the expert policies.

Got that. Thank u very much.

**Comment by RayYoh** [comment]
Hello, Sashwat. I'm sorry to bother you again. I want to ask whether you train the expert policy from scratch or ask for the model from RLAfford authors. Because I try to train it from scratch (modify the num_envs to 2, cause it will give an error for my machine if I use the original num_envs or even just use 5), but the performance seems not good.
![image](https://github.com/johnrso/spawnnet/assets/35106336/ffb23ba3-e6a0-4039-9333-2d89f40cb500)

**Comment by sashwat-mahalingam** [comment]
We obtain the expert policies from the RLAfford authors.

**Comment by RayYoh** [comment]
Hello, Sashwat. I have succeeded in reproducing your results. But I have one question. It doesn't seem to be a coincidence that several items have a maximum success rate of 0.9524, which is the same in different configurations. I also checked the log file and found that during different iterations and under various configurations, the success rate is always a set of numbers, like 0.6667, 0.8571, and 0.9048. Is there any error for my code?

![image](https://github.com/johnrso/spawnnet/assets/35106336/2f0259bf-912f-4670-b321-0627fd698747)
